.gitignore

```diff
@@ -152,11 +151,6 @@ bin/
 pkg/
 dist/
 
-
-
-# Added by cargo
-#
-# already existing elements were commented out
-
-#/target
-#Cargo.lock
+# Generated by TailwindCSS #
+############################
+src/styles/tailwind.css
```

Comment:
The output from TailwindCSS now lives in this file, so we should avoid committing it as it is generated by TailwindCSS both on development and build process